contracts/TalentLayerEscrow.sol

```diff
@@ -160,9 +160,9 @@ contract TalentLayerEscrow is Initializable, ERC2771RecipientUpgradeable, UUPSUp
      * @param _amount The amount released.
      */
     event OriginValidatedProposalFeeRateReleased(
+        address indexed _token,
```

Comment:
this reorder doesn't optimised any gas, can you tell me more of what you had in mind please?